templates/default-layout.lucius

```diff
@@ -23,6 +25,16 @@
  **     <div .my-component__item>
  **
  ** Components can be nested.
+ **
+ **
+ ** TYPOGRAPHY
+ **
+ ** Typographic choices for sizes, line-heights and margins are based on a
+ ** musical major third scale (4:5). This gives us a way to find numbers
+ ** and relationships between them that are perceived as harmonic.
+ **
+ ** To make use of this modular scale, follow this formula:
+ **   (5/4)^n | n ∈ [-6, 8]
```

Comment:
I guess it's difficult to understand without an example? Maybe we could also add this link that shows the relationship:

http://www.modularscale.com/?1&em&1.25&js&text

Or write out the actual numbers: `…, 0.64, 0.8, 1, 1.25, 1.563, …`

Should I try to add something like this? (And where, now that it's been merged (yay!)).